doc/api/http.md

```diff
@@ -777,6 +777,24 @@ const cookie = request.getHeader('Cookie');
 // 'cookie' is of type string[]
 ```
 
+### `request.getRawHeaderNames()`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {string[]}
+
+Returns an array containing the unique names of the current outgoing raw
```

Comment:
Add a comment that this is useful because it preserves casing? 

It might also make sense (again also fine for a future PR) to update the deprecation warning.

Reply:
I've included the new method inside the deprecation section.

Reply:
Also I didn't updated the documentation for this method because it feels like the documentation strives to be minimal. Also the commented line with the results after the code snippet hints at what the output is. Otherwise I agree that more information can be added as well.

Reply:
Update docs https://github.com/nodejs/node/pull/37660/commits/933100d5f1f08c4e5155f9a458a263c5f7cf7d8f, not sure if that makes sense or not.

Reply:
Good for me.